docs/troubleshooting.md

```diff
@@ -0,0 +1,61 @@
+# Troubleshooting
+
+Having a little trouble with the MFTF? See some common errors and fixes below.
+
+## WebDriver issues
+
+Troubleshoot your WebDriver issues on various browsers.
+
+### PhantomJS
+
+You are unable to upload file input using the MFTF actions and are seeing the following exception:
+
+```terminal
+[Facebook\WebDriver\Exception\NoSuchDriverException]
+No active session with ID e56f9260-b366-11e7-966b-db3e6f35d8e1
+```
+
+#### Reason
+
+Use of PhantomJS is not actually supported by the MFTF.
+
+#### Solution
+
+For headless browsing, the [Headless Chrome][]{:target="_blank"} has better compatibility with the MFTF.
```

Comment:
```suggestion
For headless browsing, the [Headless Chrome][]{:target="\_blank"} has better compatibility with the MFTF.
```
escape _ as a special symbol in markdown

Reply:
Good.